modules/home/assets/js/app.js

```diff
@@ -0,0 +1,33 @@
+import { DirectionProvider, LocalizationProvider } from '@elementor/ui';
+import { ThemeProvider } from '@elementor/ui/styles';
+import ReactUtils from 'elementor-utils/react';
+
+import HomeScreen from './components/home-screen';
+
+const App = ( props ) => {
+	return (
+		<DirectionProvider rtl={ props.isRTL }>
+			<LocalizationProvider>
+				<ThemeProvider colorScheme={ props.colorScheme }>
+					<HomeScreen />
+				</ThemeProvider>
+			</LocalizationProvider>
+		</DirectionProvider>
+	);
+};
+
+const isRTL = elementorCommon.config.isRTL,
+	rootElement = document.querySelector( '#e-home-screen' ),
+	colorScheme = rootElement?.dataset?.ui_theme || 'auto';
```

Comment:
```suggestion
	colorScheme = elementor?.getPreferences?.( 'ui_theme' ) || 'auto';
```
Maybe?

Reply:
@louiswol94 This isn't working. I think `elementor` isn't available outside of the editor. I wasn't able to get it to work.